src/phasorpy/phasor.py

```diff
@@ -279,7 +270,9 @@ def phasor_to_polar(
     >>> phasor_to_polar(
     ...     [1.0, 2.0, 3.0], [4.0, 5.0, 6.0]
     ... )
-    (array(...), array(...))
+    (array([1.326, 1.19, 1.107]),
+    array([4.123, 5.385, 6.708]))
```

Comment:
Could be on one line? `(array([1.326, 1.19, 1.107]), array([4.123, 5.385, 6.708]))`